### ISSUE TITLE
Soundtrack

### ISSUE THREAD

**Issue by Viltoncio** [xtreme8000/CavEX#31]
Hey, I recently came across your work and today I just tried it on my Wii! I never thought I would play this on the Wii! I did miss some music tho so I was thinking of making original music for the game... Do you think it's a good idea? I don't know how to code but I can come up with some music demos in about 3 days I think. No wait, I actually just did a quick demo.

https://drive.google.com/drive/folders/1CtgDzfM7Kx4rvoGusIV3Z0dGgbLlTrLl

This is my style, I just downloaded a random sf2 instrument package for LMMS so I could make this, so it's very raw

**Comment by dany-on-demand** [comment]
It sounds AI-generated

**Comment by Viltoncio** [comment]
Prolly because it's a really simple idea, no real instruments, no mixing (just effects) and it's pretty short

**Comment by madboy234** [comment]
Nice music buddy! It sounds really nice but it does still needs a little bit of improvement, like making it more calming and relaxing and some other stuff. But keep up the good work!

**Comment by Viltoncio** [comment]
yeah I think I need to slow it down a little and the ambience thing is a little more about production, I might get help with that, I only did chiptune in the past

**Comment by Viltoncio** [comment]
I was thinking, should I use Minecraft sounds instead? Like this.
Check it out @madboy234 

https://github.com/xtreme8000/CavEX/assets/94490579/de5f9491-f522-49b7-9e63-cb6e72166dca

**Comment by xtreme8000** [comment]
imo some of the notes sound way off

**Comment by Viltoncio** [comment]
@xtreme8000 C# and D clashing, but it's okay, we'll just select the best demos, I'm trying to unclash it but Idk if it works
meanwhile I have another one 

https://github.com/xtreme8000/CavEX/assets/94490579/6d99f2a3-52a0-4b7b-9e66-3b67b1c6b15f

**Comment by Viltoncio** [comment]
Btw whoever wants to try EVEN IF YOU DON'T KNOW ANYTHING ABOUT MUSIC, you can try this:
https://musiclab.chromeexperiments.com/Song-Maker/

If you mess around a bit you might get something then you can just download the midi file and import it on LMMS OR just post the link and I'll do that for you plus try to do a demo. We can choose the best ones later to work it out and polish it

**Comment by chasio2221** [comment]
this is cool
